File: examples/java-gradle-fixture/src/main/java/com/example/Calculator.java
// A minimal, deliberately clean class: builds and tests successfully, for
// `paws ci --toolchain java`'s "clean run" acceptance scenario, matching
// examples/java-maven-fixture's role for the Maven build system.
package com.example;

public class Calculator {
    public static int add(int a, int b) {
        return a + b;
    }
}

File: examples/kotlin-fixture/src/main/kotlin/com/example/Calculator.kt
// A minimal, deliberately clean object: builds and tests successfully, for
// `paws ci --toolchain kotlin`'s "clean run" acceptance scenario, matching
// examples/java-maven-fixture's role for Java.
package com.example

object Calculator {
    fun add(a: Int, b: Int): Int = a + b
}
